backend/app/services/project_automations.py

```
@@ -0,0 +1,1167 @@
+            .order_by(ProjectChatAgent.created_at)
+            .all()
+        )
+        member_rows = (
+            db.query(ResourceMember, User)
+            .join(User, User.id == ResourceMember.user_id)
+            .filter(
+                ResourceMember.resource_type == ResourceType.CLOUD_PROJECT.value,
+                ResourceMember.resource_id == event.project_id,
+                ResourceMember.entity_type == "user",
+                ResourceMember.status == MemberStatus.APPROVED.value,
+                User.is_active == True,
```

Ruff (0.16.1)

[error] Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)